Makefile

```diff
@@ -185,13 +185,13 @@ package: sdist check-sdist-local manylinux-wheels check-wheel-local
 # Build APM Python AWS lambda layer as zip artifact
 # with extension compatible with current environment
 # (x86_64 OR aarch64)
-target_dir := "./tmp"
+target_dir := "./tmp-lambda"
 aws-lambda: check-zip wrapper
 	@if [ -f ./dist/solarwinds_apm_lambda_${platform}.zip ]; then \
 		echo -e "Deleting old solarwinds_apm_lambda_${platform}.zip"; \
 		rm ./dist/solarwinds_apm_lambda_${platform}.zip; \
 	 fi
-	rm -rf ./tmp
+	rm -rf ./tmp-lambda
```

Comment:
Minor curiosity why this and a few other commands below don't use `${target_dir}` instead?

Reply:
No reason! Fixed in [29e600c](https://github.com/solarwinds/apm-python/pull/214/commits/29e600c63bef8d313117eff458c74a9e98c9dd18) 😸